<?xml version="1.0"?>
<launch>

	<!-- send ROBOT parameters to parameter server -->
	<rosparam command="load" ns="/script_server/arm_left" file="$(find cob_pushing)/robots/cob4-8/script_server/arm_left_joint_configurations.yaml"/>
	<rosparam command="load" ns="/script_server/arm_right" file="$(find cob_pushing)/robots/cob4-8/script_server/arm_right_joint_configurations.yaml"/>
	<rosparam command="load" ns="/script_server/gripper_left" file="$(find cob_pushing)/robots/cob4-8/script_server/gripper_left_joint_configurations.yaml"/>
	<rosparam command="load" ns="/script_server/gripper_right" file="$(find cob_pushing)/robots/cob4-8/script_server/gripper_right_joint_configurations.yaml"/>


</launch>
